tests/system/src/testscriptapp1/resources/mymod/myapp1/demo_01.fxml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
```

Comment:
Please add a copyright header here. See any of the other `.fxml` file in `tests/system` for an example.